README.md

```diff
@@ -21,12 +20,128 @@ Or install it yourself as:
+```json
+{
+  "items": [
+    {
+      "id": 1,
+      "name": "1c",
+      "something": 27
+    },
+    {
+      "id": 2,
+      "name": "i",
+      "something": 68
+    },
+    ...
+    {
+      "id": 25,
+      "name": "2m",
+      "something": 32
+    }
+  ],
+  "count": 25,
+  "total_pages": 2,
+  "total_count": 28,
+  "current_page": 1,
+
+}
+```
+
+Page number is passed through the `page` option of the `render_paginated` method. If none is supplied, `params[:page]` will be used (or the default parameter configured in the initializer). Default is 1.
+The amount of items is passed through the `limit` option of the `render_paginated` method. If none is supplied, `params[:limit]` will be used (or the default parameter configured in the initializer). Default is 25.
+The default serializer and formatter will be used.
+
+### Customizing output
+#### Custom serializers
+A custom serializer for each object can be passed using the `each_serializer` option:
+```ruby
+render_paginated DummyModel, each_serializer: CustomDummyModelSerializer
+```
+where the serializer is just an [`ActiveModel::Serializer`](https://github.com/rails-api/active_model_serializers).
+
+#### Custom formatters
+A formatter is an object that defines the output of the render_paginated method. In case the application needs a different format for a request, it can be passed to the `render_paginated` method using the `formatter` option:
+```ruby
+render_paginated DummyModel, formatter: CustomFormatter
+```
+or it can also be set as a default in the initializer.
+
+A new formatter can be created inheriting from the default one. The `format` method should be redefined returning something that can be converted to json.
+
+```ruby
+class CustomFormatter < Wor::Paginate::Formatter
+  def format
+    { page: serialized_content, current: current_page }
+  end
+end
+```
+
+Available helper methods are:
+* `current_page`: integer with the current page
+* `count`: number of items in the page (post-pagination)
+* `total_count`: number of total items (pre-pagination)
+* `total_pages`: number of pages given the current limit (post-pagination)
+* `paginated_content`: its class depends on the original content passed to render_paginated, it's the paginated but not yet serialized content.
+* `serialized_content`: array with all the items after going through the serializer (either the default or a supplied one)
+
+### Working with Kaminari or will_paginate
+If either Kaminari or will_paginate are required in the project, Wor::Paginate will use them for pagination with no code or configuration change.
+
+### Test helpers
+You can use the `be_paginated` matcher to test your endpoints. It also accepts the `with` chain method to receive a formatter.
+
+You only need to add this in your spec_helper.rb (or rails_helper.rb)
+```
+# spec/spec_helper.rb
+require 'wor/paginate/rspec'
+```
+And in your spec do
+```
+# spec/controllers/your_controller.rb
+RSpec.describe YourController, type: :controller do
```

Comment:
I'd replace `Rspec.describe` with `describe` and also remove the type parameter as the example is already placed into the controller's folder